src/components/OpenSource/Talks.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+
+import BlueBackground from '../Common/BlueBackground'
+import CompactVideoLink from '../Common/CompactVideoLink'
+import StyledLink from '../Common/StyledLink'
+import { Grid, Row, Col } from '../grid'
+import { SectionTitle } from '../Typography'
+
+const StyledImage = styled.img`
+  padding: ${remcalc(83)} 0 ${remcalc(15)};
+`
+
+const StyledSectionTitle = styled(Col)`
+  padding: 0 0 ${remcalc(35)};
+
+  ${breakpoint('smallTablet')`
+    padding-left: ${remcalc(21)};
+  `}
+
+  ${breakpoint('tablet')`
+    padding-left: ${remcalc(24)};
+  `}
+`
+
+const StyledTalksRow = styled(Row)`
+  padding-bottom: ${remcalc(35)};
+`
+
+const StyledTalksLink = styled(StyledLink)`
+  padding-bottom: ${remcalc(90)};
+`
+
+const TalkCol = ({
+  title,
+  type,
+  link,
+  cta,
```

Comment:
You aren't getting all of these fields from Contentful & there's no need to destructure props that you don't use anyways.